src/main/java/mate/academy/spring/service/security/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service.security;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(Role.RoleName roleName);
```

Comment:
![image](https://user-images.githubusercontent.com/55763108/236462529-ad20b1ee-906c-4797-b7cc-4df346b37160.png)
according to the requirements of the task, the input parameter must be of type String